lib/src/tag/Tag.stories.tsx

```diff
@@ -56,6 +56,36 @@ export const Chromatic = () => (
       <Title title="Icon background color" theme="light" level={4} />
       <DxcTag label="Tag" icon={icon} iconBgColor="#fabada" />
     </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Disabled" theme="light" level={4} />
+      <DxcTag label="Tag" icon={icon} iconBgColor="#fabada" disabled={true} />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Disabled with action" theme="light" level={4} />
+      <DxcTag
+        label="Tag"
+        icon={icon}
+        onClick={() => {
+          console.log("click");
+        }}
+        disabled={true}
+      />
+    </ExampleContainer>
+    <ExampleContainer pseudoState="pseudo-focus">
```

Comment:
I think this test is functional rather than visual, we should have this in the jest set of tests.

Reply:
With a disabled test for the basic tag, tag with action and tag with the link I think it's enough.

Reply:
My proposal:

- Visual tests (storybook): Disabled as action, disabled as Link. We can keep having disabled + focused visual tests
- Jest tests: disabled tag is not clickable, both as a link and as an action (button).